### ISSUE TITLE
Missing dependency on running init

### ISSUE THREAD

**Issue by rajdip-b** [tremorlabs/tremor-cli#10]
The dependency `@headlessui/tailwindcss` is not added by the `npx @tremor/cli@latest init` command. Am I doing something wrong or this is a bug?

**Comment by severinlandolt** [comment]
No, you are right here. This should be added! I am working on an improved theming structure, planning it to release it this month. Would you be open to contribute once the new structure is in place?

**Comment by rajdip-b** [comment]
> No, you are right here. This should be added! I am working on an improved theming structure, planning it to release it this month. Would you be open to contribute once the new structure is in place?

Would be happy to help!!

**Comment by severinlandolt** [comment]
Hey there @rajdip-b

Sorry for the long wait! I merged some PR's and coded up some news stuff, including the missing dependencies.

Would be great to have you take a look if it works as you expect, just run

```bash
npx @tremor/cli@beta init
```

**Comment by rajdip-b** [comment]
Great! I will surely check this out.

**Comment by severinlandolt** [comment]
@rajdip-b Did it work as expected?

**Comment by rajdip-b** [comment]
Hey @severinlandolt, sorry for the late response! I just tried it now, and it seems to work fine. Although, I would like to propose a change, if it doesn't break the flow. 

![image](https://github.com/tremorlabs/tremor-cli/assets/83924254/1a790532-0e2b-4050-bae3-1f21387b6407)

Currently, the CLI can't detect what kind of project it is init-ing in. So, it would be nice to have auto-detection feature (from package.json / next config / vite config). If the kind cant be determined, we fall back to the menu driven approach.
